### PR TITLE
MVP conditions met

### PR DESCRIPTION
**Pull request by archmagos**
All basic hashing/database functions work correctly
Branch ready for merge into master, as our back-end MVP conditions have now been met

### CODE REVIEW

File: README.md [hunk 1]
```diff
@@ -43,7 +43,6 @@ I want a confirmation with my ticket number
 | Stack    | Technologies Used | Test Suites |
 | -------- | ----------------- | ----------- |
 | BackEnd - Dummy blockchain   | Ruby, Sinatra     | Rspec       |
-| BackEnd - Blockchain         | Ethereum, Solidity     | Truffle       |
```
Comment (dct-lau17) [inline README.md]: :(

File: Rakefile [hunk 2]
```diff
@@ -10,22 +10,23 @@ end
...
-    con = PG.connect(dbname: "#{database}")
+    con = PG.connect(dbname: database.to_s)
 
     con.exec('CREATE TABLE blocks(id SERIAL PRIMARY KEY, sender CHAR(64), ' \
-             'receiver CHAR(64), value INT, previous_tx CHAR(64));')
-    con.exec('INSERT INTO blocks(sender, receiver, value, previous_tx) VALUES' \
-     "('0000000000000000000000000000000000000000000000000000000000000000'," \
-     "'0000000000000000000000000000000000000000000000000000000000000000'," \
-     "0, '0000000000000000000000000000000000000000000000000000000000000000');")
+             'receiver CHAR(64), value INT, hash CHAR(64), ' \
```
Comment (dct-lau17) [inline Rakefile]: nice refractor

File: lib/block.rb [hunk 3]
```diff
@@ -0,0 +1,66 @@
+# frozen_string_literal: true
```
Comment (dct-lau17) [inline lib/block.rb]: what does this commented line do?

File: README.md (hunk 1 above)
Comment (archmagos) [reply README.md]: 😞

File: lib/block.rb (hunk 3 above)
Comment (archmagos) [reply lib/block.rb]: Just for passing the linter

File: Rakefile (hunk 2 above)
Comment (archmagos) [reply Rakefile]: 💯

**Pull request merged**